scripts/seed-display.ts

```diff
@@ -48,42 +54,44 @@ window.onload = async () => {
 
 (window as any).goToSignIn = () => {
   setAllSeedContainersInvisible();
-  uiSeedSignIn.style.removeProperty("display");
+  uiSigninPage.style.removeProperty("display");
 };
 
 (window as any).goToSignUp = () => {
+  // Hide all containers.
   setAllSeedContainersInvisible();
 
+  // Generate the phrase.
   const generatedPhrase = generatePhrase();
-  (<HTMLInputElement>document.getElementById("signup-passphrase-text")).value = generatedPhrase;
+  uiSignupPassphraseText.value = generatedPhrase;
 
-  uiSeedSignUp.style.removeProperty("display");
+  // Show sign up container.
+  uiSignupPage.style.removeProperty("display");
 };
 
 (window as any).signIn = (event: Event) => {
   event.preventDefault();
 
-  const phraseValue = (<HTMLInputElement>document.getElementById("signin-passphrase-text")).value;
-
+  const phraseValue = uiSigninPassphraseText.value;
   if (phraseValue === "") {
     return setErrorMessage("Passphrase cannot be empty");
   }
 
   const [valid, error, seed] = validatePhrase(phraseValue);
-
   if (!valid || !seed) {
     return setErrorMessage(error);
   }
 
-  handleSeed(seed);
+  handleSeedAndEmail(seed, null);
```

Comment:
This will nullify `readEmail`. Does that need to happen? Will the email be filled out later?

Reply:
I set it to `null` here because I hadn't implemented the email field for signin yet. I'll do that now that we have an idea for the flow.

Reply:
Should we add a `TODO` and follow up with that?

Reply:
We could have, but I already implemented this flow in my next PR coming soon.